src/models/Plan.tsx

```diff
@@ -11,6 +11,14 @@ export class Plan implements SubscribableModel {
       studentId,
     });
 
+  static async doesPlanExist(studentId: string, name: string): Promise<boolean> {
```

Comment:
I prefer change the name to `isPlanExist` instead of `does` (we've already used this kind of naming)